cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,34 @@
-describe('', () => {
-  before(() => {
+const faker = require('faker');
 
-  });
-
-  it('', () => {
+describe('Conduit', () => {
+  const title = faker.lorem.word();
+  const desc = faker.lorem.sentence();
+  const body = faker.lorem.sentence();
 
+  beforeEach(() => {
+    cy.task('generateUser').then(user => {
+      cy.login(user.email, user.username, user.password);
+    });
+    cy.visit('/');
   });
 
-  it('', () => {
+  it('should provide an ability to create the article', () => {
+    cy.contains('New Article').click();
+    cy.get('[placeholder="Article Title"]').type(title);
+    cy.get('[placeholder^="What\'s this"]').type(desc);
+    cy.get('[placeholder^="Write your"]').type(body);
+    cy.contains('Publish Article').click();
+    cy.get('h1').should('contain', title);
```

Comment:
check this test, it always fails on this step 
![image](https://github.com/mate-academy/qa_cypress_e2e_article_flow/assets/85580286/efda97a8-f93f-4060-acfe-d2ccc963fee9)